redirects-www.conf

```diff
@@ -256,9 +256,9 @@ rewrite ^/pages/brochures/internetcomm.shtml /introduction-campaign-finance/unde
 rewrite ^/pages/brochures/jointfundraising.shtml /help-candidates-and-committees/joint-fundraising-candidates-political-committees/ redirect;
 rewrite ^/pages/brochures/jointfundraising_brochure.pdf /help-candidates-and-committees/joint-fundraising-candidates-political-committees/ redirect;
 rewrite ^/pages/brochures/notices.shtml /help-candidates-and-committees/advertising-and-disclaimers/ redirect;
-rewrite ^/pages/brochures/partnership.shtml /help-candidates-and-committees/candidate-taking-receipts/partnership-llc-contributions/ redirect;
-rewrite ^/pages/brochures/partnership_brochure.shtml /help-candidates-and-committees/candidate-taking-receipts/partnership-llc-contributions/ redirect;
-rewrite ^/pages/brochures/partnership_brochure.pdf /help-candidates-and-committees/candidate-taking-receipts/partnership-llc-contributions/ redirect;
+rewrite ^/pages/brochures/partnership.shtml https://www.fec.gov/help-candidates-and-committees/partnership-llc-contributions/ redirect;
+rewrite ^/pages/brochures/partnership_brochure.shtml https://www.fec.gov/help-candidates-and-committees/partnership-llc-contributions/ redirect;
+rewrite ^/pages/brochures/partnership_brochure.pdf https://www.fec.gov/help-candidates-and-committees/partnership-llc-contributions/ redirect;
```

Comment:
This works and you took out the candidate-taking-receipts part which is what was needed. Just an FYI that you don't need to add the https://www.fec.gov but it won't break anything to have it.